@css/main_style.css

```diff
@@ -303,11 +814,130 @@ div.main-body {
 }
 
 #section_2 {
-  background-image: url("../@images/section-2_background.jpeg");
   background-attachment: scroll;
   background-repeat: no-repeat;
   background-size: cover;
   height: 588px;
   padding-left: 40px;
   padding-right: 40px;
 }
+
+#about-myself-layout {
+  padding: 0 24px;
+  border-top-right-radius: 80px;
+  background: white;
+  gap: 24px;
+}
+
+#about-myself-inner-layout {
+  min-width: 327px;
+}
+
+#about-myself {
+  padding: 114px 24px 114px 24px;
+  align-items: center;
+  gap: 88px;
+  background: whitesmoke;
+}
+
+#contact-me {
+  background-image: url("../@images/contact-me_background.png");
+  background-attachment: scroll;
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 375px;
+  height: 850px;
+  padding-left: 40px;
+  padding-right: 40px;
+  flex-shrink: 0;
+}
```

Comment:
- [ ] Please adhere to the project design as much as possible, your previous code reviewer asked for the changes related to your contact form is not responsive with an image to show the problem but it seems like you didn't implement the changes and your design looks unprofessional. As per the requirement, your design should be responsive from >=375px to < 768px screen size and you can make it possible by using responsive values like `%age`. Even for your kind information by default `block` element is always 100%, So please have a look in order to see the problem in below attached image you can also see the video, in which I solved the issue very quickly. 

### Problem
![image](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/66476329/261b6bce-0b3f-4b12-b7be-55815cd5feae)

### Video

[Screencast from 15-07-23 11:00:10 PM IST.webm](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/66476329/45d1b605-96fe-4620-ba88-77cbf3078da2)

### Suggestion

```suggestion
#contact-me {
  background-image: url("../@images/contact-me_background.png");
  background-attachment: scroll;
  background-repeat: no-repeat;
  background-size: cover;
  height: 850px;
  padding: 0 40px;
}
```